src/Tribe/Views/V2/Widgets/Service_Provider.php

```diff
@@ -88,4 +89,18 @@ public function add_views( $views ) {
 
 		return $views;
 	}
+
+
+	/**
+	 * Filters the template vars for widget-specific items.
+	 *
+	 * @since TBD
+	 *
+	 * @param [type] $template_vars
+	 * @param [type] $view_interface
+	 * @return void
```

Comment:
:no_entry_sign: Function return type is void, but function contains return statement

<sub>hash: 0093da94fb83812d06f01db24c50e987</sub>